envoy/datadog_checks/envoy/data/conf.yaml.example

```diff
@@ -45,115 +45,233 @@ init_config:
-    ##   - cluster.out.alerting-event-evaluator-test.datadog.svc.cluster.local
+    ##           metrics:
+    ##           - <EXPOSED_METRIC_1>:
+    ##               name: <SENT_METRIC_1>
+    ##               type: <METRIC_TYPE_1>
+    ##           - <EXPOSED_METRIC_2>:
+    ##               name: <SENT_METRIC_2>
+    ##               type: <METRIC_TYPE_2>
+    ##
+    ##         The supported native types are `gauge`, `counter`, `histogram`, and `summary`.
+    ##
+    ## Regular expressions may be used to match the exposed metric names, for example:
+    ##
+    ##   metrics:
+    ##   - ^network_(ingress|egress)_.+
+    ##   - .+:
+    ##       type: gauge
     #
-    # included_metrics:
-    #   - cluster\.(in|out)\..*
+    # extra_metrics: []
 
-    ## @param excluded_metrics - list of strings - optional
-    ## Excludes metrics using regular expressions.
-    ## The filtering occurs before tag extraction, so you have the option
-    ## to have certain tags decide whether or not to keep or ignore metrics.
-    ## For an exhaustive list of all metrics and tags, see:
-    ## https://github.com/DataDog/integrations-core/blob/master/envoy/datadog_checks/envoy/metrics.py
+    ## @param exclude_metrics - list of strings - optional
+    ## A list of metrics to exclude, with each entry being either
+    ## the exact metric name or a regular expression.
+    ## In order to exclude all metrics but the ones matching a specific filter,
+    ## you can use a negative lookahead regex like:
+    ##   - ^(?!foo).*$
+    #
+    # exclude_metrics: []
+
+    ## @param exclude_metrics_by_labels - mapping - optional
+    ## A mapping of labels where metrics with matching label name and values are ignored. To match
+    ## all values of a label, set it to `true`.
     ##
-    ## If you surround patterns by quotes, be sure to escape backslashes with an extra backslash.
+    ## For example, the following configuration instructs the check to exclude all metrics with
+    ## a label `worker` or a label `pid` with the value of either `23` or `42`.
     ##
-    ## The example list below will exclude:
-    ##   - http.admin.downstream_cx_active
-    ##   - http.http.rds.0000.control_plane.rate_limit_enforced
+    ##   exclude_metrics_by_labels:
+    ##     worker: true
+    ##     pid:
+    ##     - '23'
+    ##     - '42'
     #
-    # excluded_metrics:
-    #   - ^http\..*
+    # exclude_metrics_by_labels: {}
 
-    ## @param cache_metrics - boolean - optional - default: true
-    ## Results are cached by default to decrease CPU utilization, at
-    ## the expense of some memory. Disable by setting this to false.
+    ## @param exclude_labels - list of strings - optional
+    ## A list of labels to exclude, useful for high cardinality values like timestamps or UUIDs.
+    ## May be used in conjunction with `include_labels`.
+    ## Labels defined in `excluded labels` will take precedence in case of overlap.
     #
-    # cache_metrics: true
```

Comment:
Is `cache_metrics` no longer supported? It looks like it's still used in the original implementation: https://github.com/DataDog/integrations-core/blob/4babc2c98a3e162a7da15121f4cd52f40e1a1d62/envoy/datadog_checks/envoy/envoy.py#L50

Reply:
I'm documenting openmetrics options only.

Reply:
Can we have the legacy options still be in `spec.yaml`, but just have them hidden? Similar to when Istio was updated to only include OpenMetricsV2 options but legacy options were still included albeit hidden.